salt/modules/salt_version.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+'''
+Access Salt's elemental release code-names.
+
+.. versionadded:: Neon
+
+Salt's feature release schedule is based on the Periodic Table, as described
+in the :ref:`Version Numbers <version-numbers>` documentation.
+
+Since deprecation notices often use the elemental release code-name when warning
+users about deprecated changes, it can be difficult to build out future-proof
+functionality that are dependent on a naming scheme that moves.
```

Comment:
Since we're just moving to a 1-year (minimum) deprecation cycle, we can probably remove this paragraph. The next paragraph may need some tweaking to make it flow better.

Reply:
Quotes from the SEP:

> Our plan is to have a typical release cadence of 3 months. We expect to have an intermediate release cadence of 4 months.

> With the increased release cadence in order to allow proper planning for feature deprecation, Salt will introduce a minimum 1-year, expected 3 major release cycle. If `neon` ships in January 2020, any new deprecations will not be removed until January 2021 at the earliest.

I do not see why the paragraph in question is no longer relevant.

Reply:
@waynew I rephrased the paragraph